modules/test/playwright/pages/portal-workflow-web/WorkflowPage.ts

```diff
@@ -47,4 +47,24 @@ export class WorkflowPage {
 				: `Success:Workflow assigned to ${asset}.`
 		);
 	}
+
+	async changeToCustomWorkflow(
```

Comment:
Hey @abesousa, what do you think about use `changeWorkflow` function instead create a similar one?